src/lib/styles/mixins/_header.scss

```diff
@@ -3,15 +3,13 @@
 @use "./media";
 
 @mixin button($background) {
-  width: var(--padding-4x);
-  min-width: var(--padding-4x);
-  height: var(--padding-4x);
+  & {
```

Comment:
Why did you move the declarations inside `& {`?

Reply:
Same as https://github.com/dfinity/gix-components/pull/471#discussion_r1703744381